fishtank/src/node.ts

```diff
@@ -16,7 +16,11 @@ export class Node {
     this.backend = cluster['backend']
   }
 
+  private get containerName(): string {
+    return this.cluster['containerName'](this.name)
```

Comment:
Can you make containerName as public function to access it `this.cluster.containerName()`

Reply:
The fact that Docker containers are used is an implementation detail. Such detail should remain hidden and not be exposed to the users. I'm thinking about adding a hidden `Naming` class so that we don't have to use private methods anymore.

Reply:
Create a linear task for Naming class to track it https://linear.app/if-labs/issue/IFL-1662/naming-class-for-hidden-container-name